<launch>
  <group ns="nanoplus">
    <node pkg="nodelet"
      type="nodelet"
      name="image_zeromq_to_ros"
      args="standalone image_zeromq_to_ros/ImageZeroMQToROS"
      output="screen">
      <param name="calibration" value="$(find image_zeromq_to_ros)/config/calib_130.yml"/>
      <param name="zmq_transport" value="tcp://127.0.0.1:12345"/>
      <param name="jpeg_decompress" value="false"/>
      <param name="image_channels" value="3"/>
      <param name="image_width" value="752"/>
      <param name="image_height" value="480"/>
    </node>
  </group>
</launch>
